plugins/restart.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha1
+kind: Plugin
+metadata:
+  name: "restart"
+spec:
+  platforms:
+  - head: https://github.com/achanda/kubectl-restart/archive/v0.0.1.zip
```

Comment:
Since you have a version tag, it would be better if you use `url` and therefore `sha256` fields.

`head` is for moving targets (like `master` branch).